Makefile

```diff
@@ -23,6 +24,7 @@ exec\:%:
 	forge script script/ExecTransaction.s.sol --$* --broadcast
 
 clean:
+	mkdir -p data
```

Comment:
but the `data` folder there right? It avoids issues if the user removes it unexpectedly right?

Reply:
Indeed, and it was removed from the repository when adding the ts script so I ran into this issue.